<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/background.jpg);
            background-repeat: no-repeat;
            background-attachment: fixed;
            /* background-size: cover; 移除不被Qt支持的属性 */</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title_label">
      <property name="styleSheet">
       <string notr="true">QLabel {
                            font-size: 40px;
                            font-weight: bold;
                            color: black;
                            }</string>
      </property>
      <property name="text">
       <string>蛏子筛选机</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="content_layout" stretch="0,1">
      <item>
       <widget class="QLabel" name="video_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>360</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_layout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>种类：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="confidence_label">
          <property name="text">
           <string>置信度：</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottom_layout">
      <item>
       <widget class="QPushButton" name="start_stop_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string>人工介入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>

   </layout>
 </widget>
     <widget class="QMenuBar" name="menubar">
        <property name="geometry">
            <rect>
            <x>0</x>
            <y>0</y>
            <width>800</width>
            <height>22</height>
            </rect>
        </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
  </widget>
 <resources/>
 <connections/>
</ui>
